dataproc_jupyter_plugin/tests/test_executor.py

```diff
@@ -129,6 +129,7 @@ async def mock_list_dag_run_task(*args, **kwargs):
             "dag_id": mock_dag_id,
             "dag_run_id": mock_dag_run_id,
         },
+        method="POST",
```

Comment:
You need to add `allow_nonstandard_methods=True,` here, and in all the similar lines below, in order to get the test to pass.